clang/docs/FunctionEffectAnalysis.rst

```diff
@@ -0,0 +1,503 @@
+========================
+Function Effect Analysis
+========================
+
+Introduction
+============
+
+Clang Function Effect Analysis is a C++ language extension which can warn about "unsafe"
```

Comment:
It is not only C++, but also works in C I think?

Reply:
May be worth documenting how to use in C, I know the syntax is a little different with the attrs